tdrs-backend/tdpservice/users/migrations/0033_user_nextgen_xid.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.11 on 2022-03-10 21:21
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0032_ocio_permissions_group'),
```

Comment:
@jorgegonzalez everything looks good here except that I'm noticing that this ocio migration results in datafiles tab being missing for different permission groups. can you confirm if this will still be an issue upon merging this PR? this is deployed here: https://tdp-frontend-a11y.app.cloud.gov/home

Reply:
I'm not sure I understand what the issue is? This PR is being made against the nextgenxms branch

Reply:
just reflecting our async conversation here for the record @jorgegonzalez:

 it looks like the 1699 bug fix has not yet been merged into the intermediary branch, which explains why that functionality is missing. I presume this will be resolved whenever the nextgen PR to merge into raft-tdp-main is opened later.